app/styles/ui/_tab-bar.scss

```diff
@@ -51,6 +51,17 @@
       margin-left: 4px;
       margin-top: 1px;
     }
+
+    .counter {
+      color: var(--tab-bar-count-color);
+      background-color: var(--tab-bar-count-background-color);
+      margin-left: 4px;
+      padding: 2px 5px;
+      font-size: var(--font-size);
```

Comment:
Can we change this to `font-size: var(--font-size-xs)`?

This will make the font size similar with other counters in the app (such as in the push/pull button):
https://github.com/desktop/desktop/blob/f8a69cad4bb3ed51810d3108b5012dcf6d331a27/app/styles/ui/toolbar/_toolbar.scss#L82

Reply:
I changed to `--font-size-xs` in 34634f2. It looks like the following.

<img width="355" alt="changed_badge_fontsize_xs" src="https://user-images.githubusercontent.com/4126644/40874281-2ba7ec24-66a8-11e8-9251-9cf9c058993b.png">